src/components/EventForm/EventForm.tsx

```diff
@@ -108,7 +108,9 @@ function EventForm({ onSubmit, ...rest }: EventFormProps) {
         teamsUrl,
       });
     } catch (error) {
-      setError(error.message);
+      if (error instanceof Error) {
+        setError(error.message);
+      }
```

Comment:
What if it's not an instance of Error? Maybe also check if it's a string and use that as the value